### PR TITLE
adding appshop text

### PR DESCRIPTION
**Pull request by zbtirrell**
See: https://central.tri.be/issues/75424

This is on hold because we are waiting for an image still

### CODE REVIEW

File: src/Tribe/App_Shop.php [hunk 1]
```diff
@@ -162,6 +162,17 @@ private function get_all_products() {
 					'image' => 'images/app-shop-eventbrite.jpg',
 					'is_installed' => class_exists( 'Tribe__Events__Tickets__Eventbrite__Main' ),
 				),
+				(object) array(
+					'title' => __( 'Image Widget Plus', 'tribe-common' ),
+					'link' => 'http://m.tri.be/19nv',
+					'description' => sprintf(
+						__( 'Take your image widgets to the next level with Image Widget Plus! We\'ve taken the simple functionality of our basic Image Widget and amped it up with several popular feature requests - multiple image support, slideshow, lightbox, and random image - all backed by a full year of premium support.', 'tribe-common' ),
+						'<a href="http://www.eventbrite.com/r/etp">',
```
Comment (borkweb) [inline src/Tribe/App_Shop.php]: I don't think this or the next line are necessary. They look like copy pasta

Comment (borkweb) [inline src/Tribe/App_Shop.php]: On that note, the `sprintf` doesn't look necessary.

Comment (zbtirrell) [inline src/Tribe/App_Shop.php]: lol. damnit.

**Comment by zbtirrell** [comment]
updated!

**Comment by barryhughes** [comment]
:8ball:

**Pull request merged**